INSTALL.md

```diff
@@ -310,6 +304,8 @@ libfaiss.so (or libfaiss.dylib)
 the executable should be linked to one of these. If you use
 the static version (.a), add the LDFLAGS used in the Makefile.
 
+** is this up-to-date? Does cmake generate dynamic libs? ** 
```

Comment:
CMake generate shared libs iff the `-DBUILD_SHARED_LIBS=ON` option is passed to the `cmake` invocation.